src/scopes/grants/index.test.js

```diff
@@ -11,10 +11,10 @@ import {
   sequelize,
 } from '../../models';
 
-const recipientOneName = faker.company.companyName();
-const recipientTwoName = faker.company.companyName();
-const recipientThreeName = faker.company.companyName();
-const recipientFourName = faker.company.companyName();
+const recipientOneName = `${faker.company.companyName()} - ${faker.animal.cetacean()} - ${faker.datatype.number()}`;
```

Comment:
Mixing companies and animals.

Reply:
Is it because company is not unique enough?

Reply:
yeah exactly